mlir/lib/Transforms/Utils/DialectConversion.cpp

```diff
@@ -29,6 +29,12 @@ using namespace mlir::detail;
 
 #define DEBUG_TYPE "dialect-conversion"
 
+/// Debugging flag. If set to "true", the materialization kind is attached as
+/// an attribute to builtin.unrealized_conversion_cast ops. This flag takes
+/// effect only if ConversionConfig::buildMaterialization = false. (Otherwise,
+/// the annotated ops are replaced with materializations.)
+static const bool kDumpMaterializationKind = false;
```

Comment:
Is the only reason for this being a constant that needs to be modified is not to bloat `ConversionConfig` or is there some other motivation?

Seems fine to me either way, but if there is a future place where it is more suitable a TODO might be useful